bag2-loader/pom.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Er missen een aantal dependencies, oa. JTS, junit-jupiter-api, commons-io, commons-lang3, 

het is niet zaligamakend (vooral bij geotools / xml en modules en database drivers die dynamisch code laden), maar:
`mvn org.apache.maven.plugins:maven-dependency-plugin:analyze -pl :bag2-loader` geeft een lijstje

```

[INFO] --- maven-dependency-plugin:3.2.0:analyze (default-cli) @ bag2-loader ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.locationtech.jts:jts-core:jar:1.18.2:compile
[WARNING]    org.junit.jupiter:junit-jupiter-api:jar:5.8.1:test
[WARNING]    commons-io:commons-io:jar:2.11.0:compile
[WARNING]    org.geotools:gt-opengis:jar:26.0:compile
[WARNING]    org.geotools:gt-metadata:jar:26.0:compile
[WARNING]    org.codehaus.woodstox:stax2-api:jar:4.2.1:compile
[WARNING]    org.apache.commons:commons-lang3:jar:3.12.0:compile
[WARNING] Unused declared dependencies found:
[WARNING]    org.geotools:gt-epsg-hsql:jar:26.0:compile
[WARNING]    org.geotools:gt-main:jar:26.0:compile
[WARNING]    org.geotools:gt-xml:jar:26.0:compile
[WARNING]    org.geotools.xsd:gt-xsd-gml3:jar:26.0:compile
[WARNING]    com.fasterxml:aalto-xml:jar:1.3.0:compile
[WARNING]    org.postgresql:postgresql:jar:42.3.1:compile
[WARNING]    net.postgis:postgis-jdbc:jar:2.5.1:compile
[WARNING]    com.microsoft.sqlserver:mssql-jdbc:jar:9.4.0.jre11:compile
[WARNING]    com.oracle.database.jdbc:ojdbc11:jar:21.3.0.0:compile
[WARNING]    org.geotools.jdbc:gt-jdbc-oracle:jar:26.0:compile
[WARNING]    org.junit.jupiter:junit-jupiter:jar:5.8.1:test
[WARNING]    org.dbunit:dbunit:jar:2.7.2:test
[WARNING]    commons-codec:commons-codec:jar:1.15:compile
[WARNING] Non-test scoped test only dependencies found:
[WARNING]    org.locationtech.jts:jts-core:jar:1.18.2:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.825 s (Wall Clock)
[INFO] Finished at: 2021-11-15T11:43:57+01:00
[INFO] ------------------------------------------------------------------------

```